src/styles/style.scss

```diff
@@ -24,23 +27,216 @@
 ******************************************************/
 
 .cards {
-  display: grid;
-  margin: 0 auto;
-  max-width: 55.375rem;
-  padding-top: 1rem;
-  margin-bottom: 5.7813rem;
-  width: 100%;
+    display: grid;
+    margin: 0 auto;
+    max-width: 55.375rem;
+    padding-top: 1rem;
+    margin-bottom: 5.7813rem;
+    width: 100%;
 
   @include media('tablet') {
-    gap: 1.5rem;
+      gap: 1.5rem;
     grid-template-columns: repeat(2, 1fr);
     margin-bottom: 2.9375rem;
   }
 
   @include media('desktop') {
-    gap: 3.75rem;
+      gap: 3.75rem;
     max-width: 67.5rem;
     margin-bottom: 4.66rem;
   }
+  
+}
+
+//--------------------UNDERSTANDING NF1--------------------
+//nf1-section
+ .nf1-margin{
+    margin-bottom: 0rem!important;
+    @media screen and (min-width: 768px) {
+        margin-bottom: 16rem!important;
+    }
+} 
+
+.nf1{
+    padding-left: 34px;
+    padding-right: 34px;
+    a{
+        padding-top: 0;
+    }
+ 
+}
+
+
+.plexiform-paragraph{
+    margin-top: 1rem;
+    @include media('tablet') {
+
+        display: flex;
+        align-items: center;
+    }
+    
+&__percentage{
+    width: 163px;
+    height: 30px;
+    margin-right: 1rem;
+    display: none;
+    
+    @include media('tablet') {
+        width: 272px;
+        height: 51px;
+        display: unset;
+    }
+}
+&__percentage-two{
+    width: 163px;
+    height: 30px;
+    margin-right: 1rem;
+    float: left;
+
+    @include media('tablet') {
+        
+        width: 272px; 
+        height: 51px;
+    }
+}
+
+&__percentage-box{
+    height: 14px;
+    @include media('tablet') {
+        display: none;
+    }
+}    
+}
+
+#nf1-box{
+    p{
+        max-width: 986px;
+    }
+}
+
+.kid-box{
+    width: 100%;
+    height: auto;
+    margin-top: 1rem;
+
+    .kid-nf1{
+            width: 279px;
+            margin: auto;
+            height: 186;
+
+        @media screen and (min-width:393px) {
+            height: 256px;
+            width: 360px;
+
+        } 
+
+    }
+    h6{
+        text-align: center;
+    }
+
+    @media screen and (min-width:768px) {
+        margin-bottom: -145px!important;
+    }
+    
+
+}
+//nf1-sectione
+
+//second-section
+.nf1-second{
+    max-width: 735px;
+    margin: auto;
+    padding-left: 10px;
```

Comment:
![image](https://user-images.githubusercontent.com/31933833/228304355-fa70f8a5-ffe6-4011-8b32-d4819a71b6d4.png)

This padding must be deleted